openedx/features/learner_analytics/__init__.py

```diff
@@ -0,0 +1,10 @@
+"""
+Learner analytics helpers and settings
+"""
+from openedx.core.djangoapps.waffle_utils import CourseWaffleFlag, WaffleFlagNamespace
+
+
+# Namespace for learner analytics waffle flags.
+WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='learner_analytics')
+
+ENABLE_DASHBOARD_TAB = CourseWaffleFlag(WAFFLE_FLAG_NAMESPACE, 'enable_dashboard_tab')
```

Comment:
@dianakhuang The flag ENABLE_DASHBOARD_TAB is not being used.  The page should return a 404 and perform no calculations if the flag is disabled.  Thanks.